lib/elixir_sense.ex

```diff
@@ -93,11 +94,11 @@ defmodule ElixirSense do
   """
   @spec definition(String.t(), pos_integer, pos_integer) :: Location.t() | nil
   def definition(code, line, column) do
-    case Source.subject(code, line, column) do
+    case Source.subject_with_position(code, line, column) do
```

Comment:
I hope that's easily switchable to Code.Fragment when it's time to merge https://github.com/elixir-lsp/elixir_sense/pull/170

Reply:
If Code.Fragment returns the same line and column, it should be fine